accounts/accounts_test.go

```diff
@@ -33,26 +34,24 @@ func TestSign(t *testing.T) {
 	am := NewManager(ks)
 	pass := "" // not used but required by API
 	a1, err := am.NewAccount(pass)
-	toSign := randentropy.GetEntropyCSPRNG(32)
```

Comment:
I don't think the tests for package accounts should test whether `crypto.Sign` works.
The idea behind the change was to remove the usage of package randentropy. I want to
remove randentropy later.